src/MerchantCenter/MerchantStatuses.php

```diff
@@ -189,9 +189,7 @@ public function maybe_refresh_status_data( bool $force_refresh = false ): void {
 		$this->refresh_presync_product_issues();
 
 		// Delete stale issues.
-		$delete_before = clone $this->current_time;
-		$delete_before->modify( '-' . $this->get_status_lifetime() . ' seconds' );
-		$this->container->get( MerchantIssueTable::class )->delete_stale( $delete_before );
+		$this->container->get( MerchantIssueTable::class )->delete_stale( $this->current_time );
```

Comment:
This change works because we aren't really passing the current time but the time when the class was initialized (before we refresh account/product issues). It makes the name of the variable a bit confusing, because if we would actually be passing the current time to the delete stale function then it could be deleting more then it should.

Reply:
Clarified the variable name in 176337c91107d169e435f6cb658e1b5e264f15f4